### DISCUSSION TITLE
Windows - Hardcoded Path for OptiX 7.3.0

### DISCUSSION THREAD

**Discussion by LuminousMonkey** [NVlabs/instant-ngp#222] [Ideas / Feature requests]
When building on Windows, the CMake file expects OptiX version 7.3.0.

However, the latest version offered is 7.4.0.
On Windows, it looks for OptiX in "C:\ProgramData\NVIDIA Corporation\OptiX SDK 7.3.0"

You can override this, by setting the enviroment variable OptiX_INSTALL_DIR

I initially thought it wasn't picking up the files because of a space in the path, so I created a symbolic link in my instant-ngp directory, and set the environment variable (using Powershell) with:

$env:OptiX_INSTALL_DIR = 'C:\Projects\instant-ngp\OptiX\'

However, just setting the environment variable without the symlink may work.

Then I ran the build instructions from the same shell without problems.

I'm not sure if the CMakeFile could be updated to automatically pick the latest OptiX SDK on a Windows system?

**Comment by Tom94** [comment]
Hi there, I agree! The current way OptiX is detected is suboptimal and it'd be nice in the future if users didn't have to manually specify the `OptiX_INSTALL_DIR` environment variable and could instead rely on automated discovery.

So far, we hadn't had the cycles to code up the corresponding automatic `cmake/FindOptiX.cmake` macro, but we welcome any additions in this direction.
